packages/state/recoil/selectors/wallet.ts

```diff
@@ -50,3 +54,73 @@ export const walletCw20BalancesSelector = selectorFamily<
       }))
     },
 })
+
+export const walletTokenStakedDenomsSelector = selectorFamily<
+  readonly string[],
+  WithChainId<{ walletAddress: string }>
+>({
+  key: 'walletTokenStakedDenoms',
```

Comment:
```suggestion
export const walletTokenDaoStakedDenomsSelector = selectorFamily<
  readonly string[],
  WithChainId<{ walletAddress: string }>
>({
  key: 'walletTokenDaoStakedDenoms',
```

can we rename this to make it a bit more clear the kinds of staked tokens?